src/main/resources/flows-config.yaml

```diff
@@ -633,4 +635,6 @@ subflows:
     entryScreen: householdIncomeByJob
     iterationStartScreen: householdIncomeWho
     reviewScreen: householdIncomeList
-    deleteConfirmationScreen: householdIncomeDeleteConfirmation
\ No newline at end of file
+    deleteConfirmationScreen: householdIncomeDeleteConfirmation
+landmarks:
+  firstScreen: howThisWorks
```

Comment:
I think this could be the first page that stores info? Like the language preference page

Reply:
Generally it's meant to be a marker for the overall first page in a flow, so that you know the starting point of the flow.

Reply:
Not that we can't change that.  Which page should we redirect to in the even we don't have an applicant id at all?